src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -14,21 +30,28 @@ export const TodoFilter = () => (
       <input
         data-cy="searchInput"
         type="text"
+        value={searchQuery}
         className="input"
         placeholder="Search..."
+        onChange={(event) => setSearchQuery(event.target.value)}
       />
       <span className="icon is-left">
         <i className="fas fa-magnifying-glass" />
       </span>
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
-        />
-      </span>
+      {searchQuery && (
+        <span className="icon is-right" style={{ pointerEvents: 'all' }}>
+          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
let's fix this eslint issue:smiley:
try to give `aria-label` to button, i think it can help